packages/block-library/src/heading/index.js

```diff
@@ -31,7 +30,7 @@ const supports = {
 export const settings = {
 	title: __( 'Heading' ),
 	description: __( 'Introduce new sections and organize content to help visitors (and search engines) understand the structure of your content.' ),
-	icon,
+	icon: 'heading',
```

Comment:
I'm pretty sure this was done on purpose. Would be good to check the original reasons for using "T". I think it's related to i18n

Reply:
See https://github.com/WordPress/gutenberg/issues/15340#issuecomment-488778130 and https://github.com/WordPress/gutenberg/issues/15340#issuecomment-488358257

Reply:
I do understand the "H" is not a i18n standard for Headings, but it does match the `H2`, `H3`, and `H4` heading sizes that are surfaced in the block toolbar. So there's some consistency there. Other apps like Dropbox Paper also use "H" for the heading level.